systemtest/src/main/java/io/strimzi/systemtest/enums/CustomResourceStatus.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.enums;
+
+public enum CustomResourceStatus {
```

Comment:
You do not really seem to pass this anywhere. You always seme to use seprately the type and status. So I wonder if it really should be two separate enums.

Reply:
So you recommend to have two enums -> `CustomResourceStatus` and `CustomResourceStatusType`? Or something like that?

Reply:
I'm thinking if we will use the `True` or `False` CR status somewhere else :smile: if not, I should keep just the CR status type

Reply:
I think the value of having both in one Enum would be if you would pass `CustomResourceStatus.Ready` through some methods. But you seem to always do only `CustomResourceStatus.Ready.getType()` or `CustomResourceStatus.Ready.getStatus()`. So it seems not needed if you anyway treat them independently.